README.md

```diff
@@ -18,6 +18,29 @@ asdf plugin-add poetry https://github.com/asdf-community/asdf-poetry.git
 Check [asdf](https://github.com/asdf-vm/asdf) readme for instructions on how to
 install & manage versions.
 
+### Overriding installer
+
+`ASDF_POETRY_INSTALL_URL` is an optional variable you can specific to point to
+the hosted installer of your choosing, e.g. `get-poetry.py` or the new `install-poetry.py`
+(compatible with [1.1.7+](https://github.com/python-poetry/poetry/releases/tag/1.1.7) and 
+[default in 1.2](https://python-poetry.org/blog/announcing-poetry-1.2.0a1/#deprecation-of-the-get-poetrypy-script)).
+
+For example, to force `install-poetry.py` on 1.1.9:
+
+```
+ASDF_POETRY_INSTALL_URL=https://install.python-poetry.org asdf install poetry 1.1.9
+```
+
+To force `install-poetry.py` for all `asdf-poetry` versions, you can export this in your
+`.bashrc`, `.zshrc`, etc:
+
+```
+export ASDF_POETRY_INSTALL_URL=https://install.python-poetry.org
+```
```

Comment:
Please remove this

Reply:
See 57b7a7e ([view](https://github.com/asdf-community/asdf-poetry/blob/57b7a7ec4dc0707923bb783159b71d6a63248652/README.md))